jovial.zsh-theme

```diff
@@ -456,7 +456,11 @@ typeset -gA jovial_affix_lengths=()
 }
 
 @jov.set-venv-info() {
-    if [[ -z ${VIRTUAL_ENV} ]]; then
+    if [ ! -z $CONDA_DEFAULT_ENV ]; then
```

Comment:
Maybe we could move checking `'base'` name related to `CONDA_DEFAULT_ENV`. 
Those modifies separates the checks for the `CONDA_DEFAULT_ENV` variable and its value being `'base'`, 
but that's unnecessary for hidden with `virtualenv` lib.

```bash
if [[ -n ${CONDA_DEFAULT_ENV} && ${CONDA_DEFAULT_ENV} != "base" ]]; then
```